charts/kvisor/templates/controller.yaml

```diff
@@ -71,21 +74,26 @@ spec:
               value: {{ if .Values.mockServer.enabled -}}
                        {{ (printf "%s:8443" (include "kvisor.castaiMockServer.service" .)) | quote }}
                      {{- else -}}
-                       {{ required "castai.grpcAddr must be provided" .Values.castai.grpcAddr | quote }}
+                       {{ .Values.castai.grpcAddr | quote }}
                      {{- end }}
             - name: CASTAI_CLUSTER_ID
-              value: {{ required "castai.clusterID must be provided" .Values.castai.clusterID | quote }}
+              value: {{  .Values.castai.clusterID | quote }}
         {{- range $key, $value := .Values.controller.extraEnv }}
             - name: {{ $key }}
               value: {{ $value }}
         {{- end }}
+        {{- if .Values.castai.apiKey }}
```

Comment:
This if would break `.Values.castai.apiKeySecretRef`.

Reply:
Added .Values.castai.enabled to be more consistent with features enablement.